client/.eslintrc.js

```diff
@@ -0,0 +1,86 @@
+module.exports = {
+    extends: ["airbnb-typescript", "next", "prettier"],
+    env: { es6: true, browser: true, node: true },
+    // plugins: ["jest", "relay"],
+    parser: "@typescript-eslint/parser",
+    parserOptions: {
+      project: "tsconfig.json",
+      tsconfigRootDir: __dirname,
+    },
+    rules: {
+      "import/no-extraneous-dependencies": [
+        "error",
+        {
+          devDependencies: [
+            "test.{ts,tsx,js,jsx}",
+            "test-*.{ts,tsx,js,jsx}",
+            "**/*{.,_}{test,spec}.{ts,tsx,js,jsx}",
+            "**/jest.config.{ts,js}",
+            "**/playwright.config.{ts,js}",
+          ],
+        },
+      ],
+      // "react/prop-types": 0, // don't need react/prop-types when components are typed with typescript
+      // "react/state-in-constructor": [1, "never"],
+      // "@typescript-eslint/no-unused-expressions": [1, { allowTernary: true }],
+      // "import/prefer-default-export": 0,
+      // "no-plusplus": 0, // not sure why this would be needed when we require semicolons
+      // "jsx-a11y/anchor-is-valid": 0, // Next.js's Link component inject the href in our <a> elements
+      // "no-restricted-syntax": 0,
+      // "import/no-extraneous-dependencies": [
+      //   "error",
+      //   { devDependencies: ["tests/**/*", "cypress/**/*"] },
+      // ],
+      "import/extensions": [
+        "error",
+        "never",
+        {
+          json: "always",
+        },
+      ],
+      "no-console": 1,
+      // "global-require": 0,
+      // "no-underscore-dangle": 0,
+      // "no-undef": 0, // This is covered by Typescript, no need for eslint to check that
+  
+      // // Rules below are code style. Prettier should deal with that instead of eslint
+      // "spaced-comment": 0,
+      // "import/newline-after-import": 0,
+      // "import/order": 0,
+      // "@typescript-eslint/lines-between-class-members": 0,
+      "no-var": 2,
+    },
+    // overrides: [
+    //   {
+    //     files: ["tests/**/*"],
+    //     extends: "plugin:jest/recommended",
+    //     rules: {
+    //       "jest/expect-expect": [
+    //         "warn",
+    //         {
+    //           assertFunctionNames: ["expect", "*.expectMutationToBeCalled"],
+    //         },
+    //       ],
+    //     },
+    //   },
+    //   {
+    //     files: ["cypress/**/*.js"],
+    //     extends: ["plugin:cypress/recommended"],
+    //     parserOptions: {
+    //       project: "cypress/tsconfig.json",
+    //     },
+    //     rules: {
+    //       "jest/valid-expect-in-promise": 0,
+    //       "jest/no-focused-tests": 2,
+    //       "promise/prefer-await-to-then": 0,
+    //       "no-unused-expressions": 0,
+    //       "cypress/no-unnecessary-waiting": 0,
+    //       "import/no-extraneous-dependencies": [
+    //         "error",
+    //         { devDependencies: true },
+    //       ],
+    //     },
+    //   },
+    // ],
```

Comment:
Are all these comments in this file meant to be enabled later? Or can we do away with them?

Reply:
These comments are leftovers from the cif/previous apps, I think we can safely delete them and re-add the individual one when we need them